pages/statistikk/om-statistikk.no.mdx

```diff
@@ -1,3 +1,14 @@
+import { Tableau } from '../../components/tableau'
+
 # Statistikk
 
-Statistikk om SDG-forskning i Norge er under arbeid, og blir lansert etter at datasettet er komplett (estimert tidlig 2023). 
\ No newline at end of file
+Statistikk om SDG-forskning i Norge er under arbeid, og blir lansert etter at datasettet er komplett (estimert tidlig 2023). 
+
+<Tableau 
+  hostUrl='https://rapport-dv.uhad.no/t/DUCT/views/SDG_statistics_2023-05/Dashboard1' 
+  height='640' 

```

Comment:
Must be set, or?

Reply:
No, but if not set it will be a bit too tall.